app/policies/dish_policy.rb

```diff
@@ -30,4 +31,8 @@ def order
   def order_by_current_user?
     order.user_id == user.id
   end
+
+  def user_debt_permitted?
+    Balance.new(user).total_debt > Money.new(-Dish::MAX_DEBT, 'PLN')
```

Comment:
So if you have to add a `-` to the `MAX_DEBT` const, maybe it should be defined already with it in the first place?